package.json

```diff
@@ -600,11 +600,14 @@
                             },
                             "logfilePath": {
                                 "type": "string",
-                                "description": "A path to a file where all brightscript console output will be written. If null or empty, file logging will be disabled."
+                                "description": "A path to a file where all brightscript console output will be written. If null or empty, file logging will be disabled.",
+                                "scope": "resource"
                             },
                             "enableDebugProtocol": {
                                 "type": "boolean",
-                                "description": "If true, the debugger will use the new BrightScript debug protocol, and will disable the telnet debugger. See this link for more details: https://developer.roku.com/en-ca/docs/developer-program/debugging/socket-based-debugger.md"
+                                "default": false,
+                                "description": "If true, the debugger will use the new BrightScript debug protocol, and will disable the telnet debugger. See this link for more details: https://developer.roku.com/en-ca/docs/developer-program/debugging/socket-based-debugger.md",
+                                "scope": "resource"
```

Comment:
`"scope": "resource"` doesn't belong in debug configuration settings. Remove this entry.

Reply:
done, thanks

Reply:
removed thanks